datareduction/datareducer.cpp

```diff
@@ -956,15 +956,15 @@ void initializeDataReducers(DataReducer * outputReducer, DataReducer * diagnosti
    for (it = P::diagnosticVariableList.begin();
         it != P::diagnosticVariableList.end();
         it++) {
-      if(*it == "EJE") {
+      // Sidestep mixed case errors
+      const std::string lowercase = boost::algorithm::to_lower_copy(*it);
```

Comment:
Does this require something like `-lboost_algorithm` in the Makefiles? Is there no std function to do that? I haven't compiled algorithm on some platforms (Hawk in particular). But that could be done of course.

Reply:
boost::algorithm::to_lower_copy(*it); is already in use in V5 in all the other datareducers, this was just where the electron branch was lagging behind. And there is no easy workaround for this. If we at some point drop using boost parameter reading then we can investigate coding the lowercase generator ourselves, but as now we need boost anyway, might as well use it.